Pipfile

```diff
@@ -0,0 +1,57 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
```

Comment:
did you convert this file for a previous file? Generally `Pipfile` should not be this detailed - that is the role of ` Pipfile.lock`.

the `Pipfile` sould generally list only the 1st level dependencies for your project and avoid listing exact versions so you can use `pipenv update` to automate updating dependencies.

Reply:
Yes it's converted from the requirements.txt file that we had before